_posts/2023-10-23-kubeflow-1.8-release.md

```diff
@@ -0,0 +1,374 @@
+
+1.8 includes new notebook images with updates to support multi-architectures including ARM and Power processors, and updates to Tensorflow, PyTorch and other packages.
+
+
+## What’s next
+
+The community continues to see a large increase in activity since the announcement that Kubeflow is now part of the CNCF Incubator. The community holds regular meetings and holds elections for the Kubeflow steering committee in the next few weeks.
+
+During the 1.8 release cycle, the community continues to work in terms of security, issue triage  and documentation. Our next releases will focus on helping build components for training LLMs and scalability.
+
+
+### How to get started with 1.8
+
+For trying out Kubeflow 1.8 we recommend our [installation page](https://www.kubeflow.org/docs/started/installing-kubeflow/) where you can choose between a selection of Kubeflow distributions. For more advanced users we recommend the manifest installation guide.   We continue to test and improve the documentation.   If you find an issue, please feel free to report and/or open a PR to help improve it for others.
+
+
+### Join the Community
+
+We would like to thank everyone for their contribution to Kubeflow 1.8, especially Daniela Plasencia for his work as the v1.8 Release Manager. As you can see, the Kubeflow community is vibrant and diverse, solving real-world problems for organizations worldwide.
+
+
+### Want to help? 
+
+Want to help? The Kubeflow community [Working Groups](https://github.com/kubeflow/community/blob/master/wg-list.md) hold open meetings and are always looking for more volunteers and users to unlock the potential of machine learning. If you’re interested in becoming a Kubeflow contributor, please feel free to check out the resources below. We look forward to working with you!
+
+
+
+* Visit our [Kubeflow website](https://www.kubeflow.org/) or [Kubeflow GitHub Page](https://github.com/kubeflow)
+* Join the [Kubeflow Slack channel](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
+* Join the [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss) mailing list
+* Attend a [weekly community meeting](https://www.kubeflow.org/docs/about/community/)
+
+
+## Component Versions
+
+
+<table>
+  <tr>
+   <td><strong>Maintainers</strong>
+   </td>
+   <td><strong>Component Name</strong>
+   </td>
+   <td><strong>Version</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>AutoML Working Group
+   </td>
+   <td>Katib
+   </td>
+   <td><a href="https://github.com/kubeflow/katib/issues/2156">v0.16.0</a>
```

Comment:
link should be update to https://github.com/kubeflow/katib/releases/tag/v0.16.0 once final release is available